beacon-chain/rpc/prysm/v1alpha1/beacon/validators.go

```diff
@@ -545,10 +545,6 @@ func (bs *Server) GetValidatorParticipation(
 		Epoch:     requestedEpoch,
 		Finalized: requestedEpoch <= cp.Epoch,
 		Participation: &ethpb.ValidatorParticipation{
-			// TODO(7130): Remove these three deprecated fields.
-			GlobalParticipationRate:          float32(b.PrevEpochTargetAttested) / float32(b.ActivePrevEpoch),
```

Comment:
@james-prysm is this used by the UI today?

Reply:
Yes this removal would break our UI currently